src/browser/README.md

```diff
@@ -0,0 +1,25 @@
+# Browser
+
+For the AWS VS Code Toolkit extension to work in the browser (eg: [vscode.dev](https://vscode.dev/)), we
+need to ensure that certain functionality which may have previously only been able to
+run in a desktop environment can additionally run in a browser environment.
+
+Functionality will not be immediately available in the browser since the underlying code will need
+to be modified to get things working. Functionality will be provided incrementally.
+
+## `package.json`
+
+We separate browser specific dependencies from the main `package.json`
+in the root to reduce complexity.
+
+These browser specific dependencies are consumed in the
+root `package.json` by using `"aws-toolkit-browser": "file:src/browser"`
```

Comment:
Should it be the other way around, i.e. browser/package.json consumes a base package.json? 

Is this only a temporary step towards that?

Reply:
This is the correct direction. The "base" package.json consumes the "browser" package.json since our base `package.json` is the only one that gets used to build the final VSIX.

Reply:
That's the inverse of this approach, right? https://github.com/aws/aws-toolkit-common/blob/147df8f44f08e081675e01cebf9f957ca9658add/lsp/core/aws-lsp-yaml-common/package.json#L10

Reply:
Yup, those have a "core" package which is consumed by other modules. But in our case everything goes in to the root `package.json` which has a dependency on the nested `src/browser/package.json`

Reply:
Ok, can you add a note about this specifically in this readme? And link to the above URL example

Reply:
Updated

Reply:
@justinmk3 updated to fix the package.json import issue